configuration/configuration.go

```diff
@@ -14,6 +14,8 @@ const (
 	Squash    = "squash"
 	NoSquash  = "no-squash"
 	SquashWip = "squash-wip"
+
+	StartCISkipCommitMessage = "mob start [ci-skip] [ci skip] [skip ci]"
```

Comment:
Feels weird that this constant is in the same place as the other 3 that share a concept. But I don't know how to solve that better now.

Reply:
option 1. Define it as variable.
option 2. Move it to main package because it is no longer used in configure package after deldeting config code.

Reply:
Or, introduce new type, something like `SquashOption`, to clarify the difference:

```
type SquashOption string

const (
	Squash    = SquashOption("squash")
	NoSquash  = SquashOption("no-squash")
	SquashWip = SquashOption("squash-wip")

	StartCISkipCommitMessage = "mob start [ci-skip] [ci skip] [skip ci]"
)
```